### ISSUE TITLE
fix zoom default

### ISSUE THREAD

**Issue by jonorthwash** [jonorthwash/ud-annotatrix#58]
Zoom width should default to no wider than window width.  See the following example of why the current behaviour is bad:
![2017-08-12-12 48 46_001](https://user-images.githubusercontent.com/963849/29242507-a4a1b364-7f5c-11e7-81cd-7acb418c63b9.png)

Might be related to #56.

**Comment by arademaker** [comment]
how to expand the zoom when editing long sentences?

**Comment by jonorthwash** [comment]
It seems to be in a fixed zoom state currently. Previously, scrolling over the graph would change the zoom level. In addition to zoom, we should also have wrapping as an option for long sentences.

**Comment by jonorthwash** [comment]
Also see #56.

**Comment by jonorthwash** [comment]
@maryszmary The default zoom level should be set to no greater than the width of the window.

**Comment by ftyers** [comment]
@jonorthwash I'm categorically opposed to wrapping, it is impossible to read/annotate. We need to come up with a better way of dealing with it, either using/improving vertical alignment or by scrolling horizontally.

**Comment by jonorthwash** [comment]
Wrapping is a feature some may want, but yeah, I have trouble reading it too, so I'm not opposed to not having it.  That can be an enhancement for someone who wants it to work on themselves at some point.  I think my suggestions in #143 are compatible with what you suggest.

**Comment by jonorthwash** [comment]
Also, the space to the left (as seen in below screenshot) is bad.

![2017-11-02-18 31 00_001](https://user-images.githubusercontent.com/963849/32353491-10d87a9c-bffc-11e7-9d82-368eaaa6d252.png)

**Comment by jonorthwash** [comment]
Same issue in Firefox.
![2017-11-02-18 39 13_001](https://user-images.githubusercontent.com/963849/32353763-2b37c306-bffd-11e7-92e7-5c80139d86d8.png)

**Comment by ftyers** [comment]
> Zoom width should default to no wider than window width. See the following example of why the current behaviour is bad:

Fixed in fdfe231, the relevant code is:

```
    if (VERT_ALIGNMENT) {
        //$("#cy").css("width", "1500px");
        $("#cy").css("width", $(window).width()-10);
        $("#cy").css("height", (length * 50) + "px");
    } else {
        //$("#cy").css("width", "1500px");
        $("#cy").css("width", $(window).width()-10);
        $("#cy").css("height", "400px");
    }

```
